app/src/main/java/ar/com/wolox/android/training/ui/viewpager/random/RandomPresenter.java

```diff
@@ -0,0 +1,24 @@
+package ar.com.wolox.android.training.ui.viewpager.random;
+
+import android.util.Log;
+
+import ar.com.wolox.android.training.model.ExampleModel;
+import ar.com.wolox.android.training.utils.UserSession;
+import ar.com.wolox.wolmo.core.presenter.BasePresenter;
+
+import java.util.Random;
+
+import javax.inject.Inject;
+
+public class RandomPresenter extends BasePresenter<IRandomView> {
```

Comment:
What is this? . I think that you don't use it anymore

Reply:
Its remove later